steps/init_env.sh

```diff
@@ -50,15 +50,15 @@ echo "HYBRIDSE_SOURCE: $HYBRIDSE_SOURCE"
 if [[ ${HYBRIDSE_SOURCE} = "local" ]]; then
```

Comment:
steps/init_env.sh and steps/setup_thirdparty.sh are still needed?

Reply:
it may still required from https://github.com/4paradigm/OpenMLDB/blob/main/.github/workflows/integration-test-src.yml. 
Mark as further work